stack.yaml

```diff
@@ -4,3 +4,5 @@ packages:
 - scalpel-core/
 - examples/
 resolver: lts-13.7
+extra-deps:
```

Comment:
What happens when scalpel is included as a dependency? Is this extra-dep implicitly carried through? Or will the user need to also explicitly add this dependency?

If they need to explicitly add it, then I think I'd like to wait for LTS to include 0.14.8 before pushing a new release with this dependency.

Reply:
It could be pushed to Hackage and then when stackage gets the new version it can be updated

Reply:
LTS Haskell 13.21 has 0.14.8 now